strings/french_replacement.json

```diff
@@ -0,0 +1,76 @@
+{
+
+    "french": {
+		"yes": "oui",
+		"no": "non",
+        "I'm": "j'", 
+        "am": "suis", 
+		"a": "un",
+        "and": "et", 
+		"the": "l'",
+		"I": "j'",
+		"for": "pour",
+		"with": "avec",
+		"of": "de",
+		
+		
+        "assistant": "ravageur",
+        "assistants": "ravageurs",
+        "baby": [
+            "enfant", 
+            "petit baguette"
+        ], 
+        "bad": "mal", 
+        "bye": [
+            "bon voyage", 
+            "adieu",
+			"au revoir"
+        ], 
+		"cake": "gateau",
+        "captain": "capitaine", 
+		"changeling": "changeur",
+        "cheese": [
+            "brie", 
+            "roquefort",
+			"camembert"
+        ], 
+        "cook": "cuisinier", 
+        "dad": "papa", 
+        "enemy": "silly english dog", 
+        "friend": "ami", 
+        "good": "bon", 
+        "greytide": "apache", 
+        "greytider": "apache",
+        "greytiders": "apaches",
+        "hello": [
+            "'allo", 
+            "bonjour"
+        ],   
+        "meat": [
+            "coq au vin", 
+            "boeuf"
+        ], 
+        "mom": "mama", 
```

Comment:
I'd change this to "maman" since "mama" can also be kid speak for "grandma"
You know, so we can have correct fake French on this atmos based space autism simulator

Reply:
Naturally; I couldn't live with myself if I led to an inaccurate fake French education.